### DISCUSSION TITLE
Module  '@supabase/supabase-js'  has no exported member QueryData

### DISCUSSION THREAD

**Discussion by aly-tq** [supabase/supabase#20160] [Questions] [answered]
Based on the documentation I'm expecting that I can use QueryData from supabase-js version: 2.38.4

https://supabase.com/docs/reference/javascript/release-notes#response-types-for-complex-queries

**Comment by GaryAustin1** [comment, marked as answer]
supabase-js is at 2.39.2.  I don't know when it was added, but it is very recent.

**Comment by cain** [comment]
I also had same issue, i updated supabase-js version to 2.39.2 and fixed the issue

**Comment by aly-tq** [comment]
resolved, thanks!
